.github/workflows/pr-publish-docsite.yaml

```diff
@@ -11,8 +11,48 @@ on:
   workflow_dispatch:
 
 jobs:
+  cypress:
+    name: Cypress Tests
+    runs-on: ${{ matrix.os }}
+    container:
+      image: cypress/browsers:node16.13.2-chrome97-ff96
+      options: --user 1001 --shm-size=2g     
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        browser: [chrome, firefox]
+    steps:
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - run: node -v   
+    
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Cypress run 
+        uses: cypress-io/github-action@v4.2.0
+        timeout-minutes: 40
+        with:
+          record: true
```

Comment:
@brianluisgomez Do we record just on failures? What is our purge strategy for recorded data?

Reply:
This record is what is putting the records in the dashboard so it records all tests that ran, whether passed or failed. We're only saving screenshots on failed tests but everything is recorded. Runs can be archived but not really deleted as the number of records/month looks to be how Cypress bills. We may want to look into this more to verify pricing on records per month in the org before moving forward to make sure we aren't passing any limit without realizing it.

Reply:
Let's do some analysis on understanding pricing for our usage.